go.rst

```diff
@@ -287,9 +287,9 @@ Wahl stehen.
 
 8. Wahl durch Zustimmung ist durch den folgenden Algorithmus definiert:
 
-   - Jeder wahlberechtigte Person erhält einen Wahlzettel mit einer
+   - Jede wahlberechtigte Person erhält einen Wahlzettel mit einer
      Liste aller zur Wahl stehenden Personen.
-   - Jeder zur Wahl stehenden Person kann eine Stimme gegeben werden.
+   - Jede zur Wahl stehenden Person kann eine Stimme gegeben werden.
```

Comment:
Da hast du recht, deshalb ist das in #20 auch gefixt. Ich sollte das der Klarheit evtl. hier rein mergen oder dieses PR nochmal neu bauen ohne die Änderungen aus #20.